frontend/server/api/auth/[...].ts

```diff
@@ -0,0 +1,70 @@
+// file: ~/server/api/auth/[...].ts
+import CredentialsProvider from 'next-auth/providers/credentials'
+import { NuxtAuthHandler } from '#auth'
+
+interface Customer {
+	email: string
+	id: string
+	firstName: string
+	lastName: string
+	netWorth: string
+}
+
+async function getCustomer(email: string) {
+	return await $fetch<Customer>('/api/auth/customer', {
+		method: 'POST',
+		body: {
+			email: email,
+		},
+	})
+}
+
+export default NuxtAuthHandler({
+	providers: [
+		// @ts-expect-error You need to use .default here for it to work during SSR. May be fixed via Vite at some point\
```

Comment:
Auth is not working without this line as it produces error, I don't see an issue keeping this